kitsune/search/v2/search.py

```diff
@@ -55,7 +55,13 @@ def get_highlight_fields(self):
 
     def get_filter(self):
         filters = [
+            # restrict to the question index
             DSLQ("term", _index=self.get_index()),
+            # only return questions created in the last two years
+            DSLQ(
+                "range",
+                question_created={"gte": datetime.now(timezone.utc) - timedelta(days=365.25 * 2)},
```

Comment:
let's use a variable for the two years. also why float?

Reply:
It's the mean number of days in a year :P

Reply:
cool, let's make it an integer. floor or ceil totally up to you